share/jupyter/hub/templates/admin.html

```diff
@@ -76,6 +79,10 @@
   Are you sure you want to stop all your users' servers? Kernels will be shutdown and unsaved data may be lost.
 {% endcall %}
 
+{% call modal('Start All Servers', btn_label='Start All', btn_class='btn-success start-all-button') %}
+  Are you sure you want to start all servers? This can slam your server resources.
```

Comment:
~~slam~~ suggestions: burden or overload your available server resources.

Reply:
Definitely not my best copy writing :)